src/depthMapEntity/plugin.hpp

```diff
@@ -13,7 +13,12 @@ class DepthMapEntityPlugin : public QQmlExtensionPlugin
     Q_PLUGIN_METADATA(IID "depthMapEntity.qmlPlugin")
 
 public:
-    void initializeEngine(QQmlEngine* engine, const char* uri) override {}
+    void initializeEngine(QQmlEngine* engine, const char* uri) override
+    {
+        // Fix "unused parameter" warnings; should be replaced by [[maybe_unused]] when C++17 is supported
```

Comment:
the alternative solution is to avoid putting a name in the arguments, like
like `void initializeEngine(QQmlEngine*, const char*) override {}`